redash/metrics/celery.py

```diff
@@ -1,17 +1,13 @@
-from __future__ import absolute_import
-
 import logging
 import socket
 import time
 
-from celery.signals import task_postrun, task_prerun
 from redash import settings, statsd_client
 from redash.utils import json_dumps
 
 tasks_start_time = {}
 
 
-@task_prerun.connect
```

Comment:
Why move this? It's better to have this module self contained.

Reply:
Mostly because I wanted to prevent import time side-effects. The decorators were called by importing the `redash.metrics.celery` module in `redash/worker.py` which felt like bad practices especially since we're connecting another signal handler in `worker.py` already. In other words, I think it's cleaner in this case to keep the handlers separated, but all Celery initialization in `worker.py`.

Reply:
But now if we want to remove this module we need to do cleanup in two places, instead of just removing the module import. I never checked what connecting signals in Celery does, but I'm not sure I would qualify this as side effects.